container/conf.d/daemon.conf

```diff
@@ -0,0 +1,18 @@
+[daemon]
+systemd_enable = false
+
+[tls]
+enable = false
```

Comment:
This is temporary until we have configure PKI via files mounted files in the container, but it can also be used if the server is behind a reverse proxy (e.g. k8s ingress) that will use https and communicate via http with the node.